ado2gh_migration_scripts/MigrationWithoutAssigningTeam/checkAfterMigration/checker.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd 
+import os 
+
+c_dir = os.path.dirname(os.path.realpath(__file__))
+## dependencies on the GetOrganizationRepositoriesFromGithub
+path = f'{c_dir}/orgRepositories.csv'
+migrated_df = pd.read_csv(path)
+migrated_df['full_name'] = migrated_df['full_name'].str.split('/').str[1]
```

Comment:
What is the purpose of this line of code can you please add some comments here.

Reply:
Ok I will